src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -720,15 +720,15 @@ static void window_scenery_dropdown(rct_window* w, rct_widgetindex widgetIndex,
 
     if (widgetIndex == WIDX_SCENERY_PRIMARY_COLOUR_BUTTON)
     {
-        gWindowSceneryPrimaryColour = (uint8_t)dropdownIndex;
+        gWindowSceneryPrimaryColour = static_cast<uint8_t>(dropdownIndex);
```

Comment:
gWindowSceneryPrimaryColour and friends are `colour_t`, so I'd like all three casts to cast to `colour_t` instead.